ui-participant/src/hub/HubPage.tsx

```diff
@@ -34,7 +45,27 @@ export default function HubPage() {
 
   const hubUpdate = useHubUpdate()
   const [showMessage, setShowMessage] = useState(true)
-  const hasActiveTasks = enrollees.some(enrollee => enrollee.participantTasks.some(task => isTaskActive(task)))
+  const hasActiveTasks = activeEnrollee?.participantTasks.some(isTaskActive)
+
+  function getTotalTasks() {
+    let totalTasks = 0
+    enrollees.forEach(enrollee => {
+      enrollee.participantTasks.forEach(participantTask => {
+        if (participantTask.status != 'COMPLETE') {
+          totalTasks++
+        }
+      })
+    })
+
+    relations?.forEach(enrolleeRelation => {
+      enrolleeRelation.relation.targetEnrollee.participantTasks.forEach(participantTask => {
+        if (participantTask.status != 'COMPLETE') {
+          totalTasks++
+        }
+      })
+    })
+    return totalTasks
```

Comment:
Another way to write this that's a little bit more idiomatic:

```suggestion
    const pendingEnrolleeTasks = enrollees.flatMap(enrollee =>
      enrollee.participantTasks.filter(participantTask =>
        participantTask.status != 'COMPLETE'
      )
    )

    const pendingRelationTasks = relations?.flatMap(enrolleeRelation =>
      enrolleeRelation.relation.targetEnrollee.participantTasks.filter(participantTask =>
        participantTask.status != 'COMPLETE'
      )
    )

    return pendingEnrolleeTasks.length + pendingRelationTasks?.length
```